featurebyte/query_graph/sql/interpreter/tile.py

```diff
@@ -74,6 +75,17 @@ class TileGenSql:
     parent: str | None
     agg_func: str | None
 
+    @property
```

Comment:
Do we need this property? It does not seem to be used

Reply:
It's used in a few places: 

<img width="818" alt="image" src="https://github.com/user-attachments/assets/a09bb3f3-48ac-4838-bc60-4d5c5b2636f1">